catalog/migrate/workspace-query-separation/pom.xml

```diff
@@ -16,7 +16,7 @@
   <parent>
     <artifactId>migrate</artifactId>
     <groupId>ddf.catalog</groupId>
-    <version>2.22.0-SNAPSHOT</version>
```

Comment:
odd that this wasn't updated by the release plugin

Reply:
this look like an orphan module.. should it be deleted?

Reply:
i'm not sure it should be deleted.  looks like @rzwiefel orphaned it in this PR https://github.com/codice/ddf/pull/5713/files#diff-14e72839cb14127a65766ef968658b39L29